report/common.py

```diff
@@ -0,0 +1,455 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Common libraries for report generation."""
+
+import dataclasses
+import io
+import json
+import logging
+import os
+import re
+from typing import List, Optional
+
+import yaml
+from google.cloud import storage
+
+import run_one_experiment
+from experiment import evaluator
+from experiment.workdir import WorkDirs
+
+MAX_RUN_LOGS_LEN = 16 * 1024
+
+TARGET_EXTS = ('.c', '.cc', '.cpp', '.cxx', '.c++', '.java', '.py')
```

Comment:
Sorry I missed this last time: Would it be a good idea to import [this](https://github.com/google/oss-fuzz-gen/blob/main/data_prep/project_src.py#L30) and append `['.java', '.py']`?
So that we only define it once (to ensure it is consistent across the project and simplify future changes).

Reply:
Done!